rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def intialize(date)
```

Comment:
- [ ] Great job so far @MuhammadUsmanAslam There are just a few changes you need to make and you'll be on your way to the next project 👍 
- [ ] Firstly, the initialize method is missing the `book` and `person` arguments, preventing the rental object from being properly associated with a `book` and a `person`. 😉 
- [ ]  Additionally, the spelling of the `initialize` method appears to be incorrect, which can lead to unexpected behavior. So kindly fix that as well 👍 
- Example code
` 
  def initialize(date, book, person)
`

- [ ] Aside this, I will suggest that by you add the rental object to the rentals collection of the associated `book` and `person` that way, a proper `many-to-many relationship` is established, ensuring consistency and data integrity. 👍 
- This achieves the desired `belongs-to` association between` Rental and Book/Person.`. 😉 
- Example Code

```ruby
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self
```